renku/core/models/datasets.py

```diff
@@ -44,6 +45,32 @@
 )
 
 
+def _convert_creators(value):
+    """Convert creators."""
+    if isinstance(value, dict):  # compatibility with previous versions
+        return [Person.from_jsonld(value)]
+
+    if isinstance(value, list):
+        return [Person.from_jsonld(v) for v in value]
+
+
+@attr.s
+class PersonMixin:
+    """Mixin for handling creators container."""
```

Comment:
Maybe this should still be named `CreatorsMixin` as it is for setting the creator on an object. Especially given the docstring there.

Reply:
good point - I changed it back